images/Dockerfile.tomcat

```diff
@@ -1,41 +1,6 @@
-FROM ansible/ubuntu-ssh
+FROM harishanand95/ansible-tomcat
 MAINTAINER Harish Anand "https://github.com/harishanand95"
```

Comment:
`harishanand95/ansible-tomcat` image will contain the tomcat installation steps that were part of the old Dockerfile. This avoids the manual building process that keeps happening. 
The current approach will be to download the image at once and reuse it. 
(instead of building from the Dockerfile)